ios/PurchasesHybridCommon/PurchasesHybridCommon/Purchases+HybridAdditions.swift

```diff
@@ -36,13 +36,18 @@ import RevenueCat
         if let appUserID = appUserID {
             configurationBuilder = configurationBuilder.with(appUserID: appUserID)
         }
-        configurationBuilder = configurationBuilder.with(purchasesAreCompletedBy: purchasesAreCompletedBy)
+        if let storeKitVersion {
+            if let version = StoreKitVersion(name: storeKitVersion) {
+                configurationBuilder = configurationBuilder.with(
+                    purchasesAreCompletedBy: purchasesAreCompletedBy,
+                    storeKitVersion: version
```

Comment:
The current implementation means it wouldn't work if the caller only passed a value for `purchasesAreCompletedBy` but passed `nil` for `storeKitVersion`. Can those be separated?

Reply:
The reason that these are bundled is that they're bundled in the native iOS SDK's configuration options when `purchasesAreCompletedBy` is provided:

```swift
@objc public func with(
            purchasesAreCompletedBy: PurchasesAreCompletedBy,
            storeKitVersion: StoreKitVersion
        ) -> Configuration.Builder {
```

The way it boils down, you can provide a `StoreKitVersion` without `PurchasesAreCompletedBy`, but if you provide a `PurchasesAreCompletedBy`, then you _must_ provide a `StoreKitVersion` 😅